pkg/oauthflow/interactive.go

```diff
@@ -53,16 +57,22 @@ func (i *InteractiveIDTokenGetter) GetIDToken(p *oidc.Provider, cfg oauth2.Confi
 		return nil, err
 	}
 
-	authCodeURL := cfg.AuthCodeURL(stateToken, append(pkce.AuthURLOpts(), oauth2.AccessTypeOnline, oidc.Nonce(nonce))...)
-	fmt.Fprintf(os.Stderr, "Your browser will now be opened to:\n%s\n", authCodeURL)
-	if err := open.Run(authCodeURL); err != nil {
+	opts := append(pkce.AuthURLOpts(), oauth2.AccessTypeOnline, oidc.Nonce(nonce))
+	authCodeURL := cfg.AuthCodeURL(stateToken, opts...)
+	var code string
+	if err := open.Run(authCodeURL); err == nil {
```

Comment:
```suggestion
	if err := open.Run(authCodeURL); err != nil {
```

Reply:
Nice catch! Left over from my final testing to simulate a failure :)